website/content/docs/concepts/security/data-encryption.mdx

```diff
@@ -13,39 +13,42 @@ Systems](/docs/configuration/kms) that protect the base encryption keys used
 for various functions. This page describes the various KMS key purposes that
 Boundary supports and how they are used within the system.
 
-~> **Note:** Boundary does not currently support rotation of its internal keys, but this
-will be supported in a future version. External keys can be rotated so long as
-the original keys remain available for decryption; full support for rotating
-these will come in a future version as well.
-
 ## The `worker-auth-storage` KMS Key
 
 The `worker-auth-storage` KMS key is used by a [PKI
 Worker](/docs/configuration/worker/pki-worker) for storage of authentication
 keys. It is optional for PKI workers; if not specified the authentication keys
 will not be encrypted on disk. This is not used by KMS workers.
 
-## The `root` KMS Key and Per-Scope KEK/DEKs <sup>OSS Only</sup>
+## The `rootKey` KMS Key and Per-Scope KEK/DEKs <sup>OSS Only</sup>
```

Comment:
This is the purpose name we use internally, so it's what's visible to the user on the API.